stoplight.js

```diff
@@ -0,0 +1,30 @@
+/*var redLight = $("#red");
+var redLight =button.on("click", function changeBackgroundColorToRed(){
+})
+
+var yellowLight = $("#yellow");
+var yellowLight = button.on("click", function changeBackgroundColorToYellow(){
+})
+
+var greenLight = $("#green");
+var greenLight = button.on("click", function changeBackgroundColorToGreen(){
+})*/
+
+$(function() {
+$('#red').click(function(){
+    $(this).css('background', 'red');
+
+  });
+});
```

Comment:
Also, you didn't need to set the background to red. In the stylesheet, you'll notice some classes that care of that for you. You just need to make sure to toggle the correct classes on/off, depending on the current state of the stoplight.